cypress/support/components/EditDialogModal.js

```diff
@@ -249,10 +249,12 @@ export function VerifyEntityLabeledDifferentPopupAndAccept(textEntityPairs) { Ve
 
 function VerifyEntityLabeledDifferentPopupAndClickButton(textEntityPairs, buttonLabel) {
   cy.Get('.ms-Dialog-main')     // This returns multiple parent objects
-    .contains('Entity is labelled differently in another user utterance') // Narrows it down to 1
-    .parents('.ms-Dialog-main') // Back to the single parent object
+    .contains('Inconsistent Entity Labels') // Narrows it down to the one we want
```

Comment:
Would seem better to change this selector to use the `[data-testid= ]` like this rest of selectors so it doesn't need to match the title.

Reply:
Good point, I was able to significantly simplify this method. When I originally wrote it that data-testid was not there, and I could not add one or could not get it right (don't remember now). But now that it has changed, and you added that in I was able to make it better. Thanks.

Reply:
The fix will be in another PR since this one has been pushed to master.